### PR TITLE
Import legacy frontend

### PR DESCRIPTION
**Pull request by iamdey**

### CODE REVIEW

**Comment by gdchamal** [comment]
LGTM

**Comment by iamdey** [comment]
erf commit author failed. Dunno how to fix author in that case, I restart from scratch, that would be faster

File: src/frontend/web_application/.gitignore [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# dependencies
```
Comment (iamdey) [inline src/frontend/web_application/.gitignore]: Don't you think it could be enough to add "per folder" specific rules ?

**Comment by iamdey** [comment]
new PR #53

**Pull request closed without merging**